src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,88 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './FindMovie.scss';
+import classNames from 'classnames';
+import { Movie } from '../../types/Movie';
+import { getMovie } from '../../api';
+import { MovieCard } from '../MovieCard';
+
+type Props = {
+  searchText: string;
+  setSearchText: (text: string) => void;
+  reset: () => void;
+  addMovie: (movie: Movie) => void;
+};
+
+const defaultPicture = (
+  'https://via.placeholder.com/360x270.png?text=no%20preview'
+);
+
+export const FindMovie: React.FC<Props> = ({
+  searchText,
+  setSearchText,
+  reset,
+  addMovie,
+}) => {
+  const [movie, setMovie] = useState<Movie | null>(null);
+  const [isError, setIsError] = useState(false);
+  const [isLoading, setIsLoading] = useState(false);
+
+  const eventChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const { value } = e.target;
+
+    if (value.trim().length === 0) {
+      setSearchText('');
+
+      return;
+    }
+
+    value.trim();
+    setSearchText(value);
+    setIsError(false);
+  };
+
+  const handleClick = (foundMovie: Movie) => {
+    reset();
+    addMovie(foundMovie);
+    setMovie(null);
+  };
+
+  const searchMovie = async (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    setIsLoading(true);
+    const movieFromServer = await getMovie(searchText);
+
+    if ('Error' in movieFromServer) {
+      setIsError(true);
+    } else {
+      const {
+        Title,
+        Plot,
+        Poster,
+        imdbID,
+      } = movieFromServer;
+
+      const newMovie = {
+        title: Title,
+        description: Plot,
+        imgUrl: Poster !== 'N/A'
+          ? Poster
+          : defaultPicture,
```

Comment:
handle truth case first
```suggestion
        imgUrl: Poster === 'N/A'
          ? defaultPicture 
          : Poster,
```